src/Rule/isUrl.php

```diff
@@ -0,0 +1,30 @@
+<?php
+
+namespace Morebec\Validator\Rule;
+
+
+use Morebec\Validator\ValidationRuleInterface;
+
+class isUrl implements ValidationRuleInterface
+{
+
```

Comment:
Every rule should support a custom message passed to the constructor as arguments.
Please add it here as well.